cmd/dtab.go

```diff
@@ -94,6 +94,80 @@ var (
 		},
 	}
 
+	dtabVersionCmd = &cobra.Command{
+		Use:     "version [name]",
+		Short:   "Get a version by name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			switch len(args) {
+			case 1:
+				ctl, err := getController()
+				if err != nil {
+					return err
+				}
+				name := args[0]
+				vd, err := ctl.Get(name)
+				if err != nil {
+					return err
+				}
+
+				fmt.Println(vd.Version)
+
+				return nil
+
+			default:
+				return errors.New("get requires a name argument")
+			}
+		},
+	}
+
+	dtabSetCmd = &cobra.Command{
+		Use:     "set [name] [prefix] [destination]",
+		Short:   "Update a dtab destination by name",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			switch len(args) {
+			case 3:
+				ctl, err := getController()
+				if err != nil {
+					return err
+				}
+				name := args[0]
+				vd, err := ctl.Get(name)
+				if err != nil {
+					return err
+				}
+
+				prefix := args[1]
+				destination := args[2]
+
+				found := false
+				for _, dtab := range vd.Dtab {
+					if dtab.Prefix == "/" + prefix {
+						dtab.Destination = destination
+						found = true
+					}
+				}
```

Comment:
Yes, the serverside namerd API is changing (to be primarily gRPC, though we'll likely need to maintain the plain-old-HTTP interface for the time being as well.

If your curious, here's the initial review for the read side of the API https://github.com/linkerd/linkerd/pull/1065 (effectively replacing the thrift interface for namerd).

I expect that namerctl will be moved onto the gRPC interface (once a write API is available). We're still figuring out what this new write API is going to look like, but we want to support a number of use cases that are not easy to accommodate with the current Big Flat Dtab model; so it's possible that the write API may change substantially.